precompiles/multi-asset-delegation/src/lib.rs

```diff
@@ -140,12 +138,33 @@ where
 	#[precompile::public("executeWithdraw()")]
 	fn execute_withdraw(handle: &mut impl PrecompileHandle) -> EvmResult {
```

Comment:
Don't see tests of this new logic?

Reply:
https://github.com/tangle-network/tangle/blob/578064e767a330604f2193359e724e6b10781279/precompiles/multi-asset-delegation/src/tests.rs#L245

And 

https://github.com/tangle-network/tangle/blob/578064e767a330604f2193359e724e6b10781279/precompiles/multi-asset-delegation/src/tests.rs#L314